Helpers/globalState.js

```diff
@@ -0,0 +1,67 @@
+export function updateGlobalState(state) {
+  let oldState = localStorage.getItem("state")
+
+  oldState = JSON.parse(oldState)
+
+  let newState = {
+    ...oldState,
+    ...state,
+  }
+
+  localStorage.setItem("state", JSON.stringify(newState))
+}
+
+export function updateEditDataById(id, state) {
```

Comment:
ditto (meaning do the same thing here)

`export const updateEditDateById = (id, state) => { .....
}`